src/Gui/Application.h

```diff
@@ -154,7 +157,7 @@ class GuiExport Application
     void slotResetEdit(const Gui::ViewProviderDocumentObject&);
 
 public:
-    /// message when a GuiDocument is about to vanish
+    /// message when a GuiDocumColorent is about to vanish

```

Comment:
You probably didn't mean to make this change 😄

Reply:
Yep. Middle mouse click on Linux can be a pain. :laughing: Just don't know how I could have missed it when looking at the diff...